test/automation/src/sql/notebook.ts

```diff
@@ -22,6 +22,9 @@ export class Notebook {
 	async openFile(fileName: string): Promise<void> {
 		await this.quickAccess.openQuickAccess(fileName);
 		await this.quickInput.waitForQuickInputElements(names => names[0] === fileName);
+		this.code.isActiveElement('.quick-input-widget .quick-input-box input').catch(e => {
```

Comment:
Probably fine to just await the call to isActiveElement here and let the test fail if this isn't the active element

Reply:
When I remove the catch, I get this error:

(node:90722) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)

and the active element error message is not shown. I'll just leave the catch since the test will fail in the next step if the current element isn't the input dialog.

Reply:
You should be awaiting the call - it shouldn't be unhandled then

Reply:
`await this.code.isActiveElement('...');`

Reply:
Yep I am awaiting the call but still getting that error.

Reply:
uh, that's...interesting. How are you getting this error? Are you throwing a test error or something?

Reply:
I'm changing what's being passed into isActiveElement() to something else so that the error gets thrown.

Reply:
This is the whole message I see:

(node:94121) UnhandledPromiseRejectionWarning: TypeError: input.replace is not a function
    at MochaJUnitReporter.removeInvalidCharacters (/Users/lucyzhang/GitProjects/azuredatastudio/test/smoke/node_modules/mocha-junit-reporter/index.js:362:16)
    at MochaJUnitReporter.getTestcaseData (/Users/lucyzhang/GitProjects/azuredatastudio/test/smoke/node_modules/mocha-junit-reporter/index.js:346:20)
    at MochaJUnitReporter.<anonymous> (/Users/lucyzhang/GitProjects/azuredatastudio/test/smoke/node_modules/mocha-junit-reporter/index.js:234:27)
    at Runner.emit (events.js:203:15)
    at Runner.fail (/Users/lucyzhang/GitProjects/azuredatastudio/test/smoke/node_modules/mocha/lib/runner.js:304:8)
    at /Users/lucyzhang/GitProjects/azuredatastudio/test/smoke/node_modules/mocha/lib/runner.js:673:18
    at done (/Users/lucyzhang/GitProjects/azuredatastudio/test/smoke/node_modules/mocha/lib/runnable.js:334:5)
    at /Users/lucyzhang/GitProjects/azuredatastudio/test/smoke/node_modules/mocha/lib/runnable.js:398:11
    at process._tickCallback (internal/process/next_tick.js:68:7)
warning.js:25
(node:94121) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
warning.js:25
(node:94121) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

Reply:
What happens if you change it to an await and then change 335 in the other file to 

`return await this.driver.isActiveElement(windowId, selector);`?

It's weird that the mocha code is also throwing an error so not sure what's going on here.

Reply:
Still seeing the same error with those changes

Reply:
Can you push your changes? I don't see how this is possible - all the other calls are doing pretty much exactly the same thing.

Reply:
Just pushed changes. Yeah I'm not sure what's happening - all the other methods like waitForElement will throw an error here: https://github.com/microsoft/azuredatastudio/blob/452a0d2990fd139b19703565ccc7fd8691ee3b41/test%2Fautomation%2Fsrc%2Fcode.ts#L213

Reply:
You know - stepping back a bit what if we just did waitForActiveElement on the input box. That should accomplish the same thing and doesn't need us to change anything. (adding the current active element to the error message though would be useful)

Reply:
Right, waitForActiveElement wouldn't give us the current active element.

Reply:
I think it's reasonable to add the current active element (from the error message of isActiveElement) to the error message there

Reply:
Ah turns out I was wrong - waitForActiveElement calls poll and poll actually logs the last error thrown by isActiveElement. https://github.com/microsoft/azuredatastudio/blob/452a0d2990fd139b19703565ccc7fd8691ee3b41/test%2Fautomation%2Fsrc%2Fcode.ts#L211
Which means we can find the current active element in our logs if waitForActiveElement fails. 
I'll update my PR, thanks Charles!